controllers/managers.js

```diff
@@ -83,3 +87,60 @@ exports.confirm = async (req, res) => {
             });
         });
 };
+
+// @desc    Login a manager
+// Route    POST /api/v1/managers/login
+// Access   Public
+exports.login = async (req, res) => {
+    const { email, password } = req.body;
+
+    db.sync({ logging: false })
+        .then(async () => {
+            try {
+                let token;
+                const manager = await Manager.findOne({ where: { email } });
+
+                if (!manager.dataValues) {
+                    throw new Error(`Manager doesn't exist!`);
+                }
+                const isValid = await bcrypt.compare(
+                    password,
+                    manager.dataValues.password
+                );
+                if (isValid) {
+                    token = jwt.sign(
```

Comment:
another util function could also be awsome in the future as it could also be use to sign the email confirmation token